iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -137,6 +146,7 @@
 
 public class HiveIcebergMetaHook implements HiveMetaHook {
   private static final Logger LOG = LoggerFactory.getLogger(HiveIcebergMetaHook.class);
+  private final ObjectMapper jsonObjectMapper = new ObjectMapper();
```

Comment:
see https://stackoverflow.com/questions/3907929/should-i-declare-jacksons-objectmapper-as-a-static-field
````
keep static ObjectMapper somewhere, but only get ObjectReader / ObjectWriter instances (via helper methods), retain references to those in other places (or dynamically call). These reader/writer objects are not only fully thread-safe wrt reconfiguration, but also very light-weight (wrt mapper instances). So keeping thousands of references does not add much memory usage
````